### ISSUE TITLE
Question on discriminator labels

### ISSUE THREAD

**Issue by maderafunk** [JGuillaumin/SuperResGAN-keras#1]
Hi,
you have a very nice collection of models, it has been very helpful for me. I was wondering, however, what is the motivation to choose labels between 0.7 and 1.2 in your training loop for the discriminator? Doesn't the sigmoid function scale the output between 0 and 1, so wouldn't it make sense to clip the labels at 1.0?
                         
  ```
                                  #On real/HR images#
    batch_LR, batch_HR = batch_gen.next()
    # training on noisy labels
    real_labels = np.random.uniform(0.7, 1.2, size=batch_HR.shape[0]).astype(np.float32)
```
